docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md

```diff
@@ -40,6 +41,22 @@ These suffixes can also take modifiers themselves. The modifiers currently suppo
 comparison uses the database's default. For MySQL and MSSQL, this is case-insensitive. For PostgreSQL, this is 
 case-sensitive.
 
+```php
+// Fetch players that their FirstName is 'Sam' or 'sam' or 'sAm'... (NOT case-sensitive)
+$players = Player::get()->filter([
+    'FirstName:ExactMatch' => 'Sam'
+]);
+
+// Fetch players that their FirstName is 'Sam' (case-sensitive)
+$players = Player::get()->filter([
+    'FirstName:ExactMatch:case' => 'Sam'
+]);
+
+// Shorthand versions of the above:
```

Comment:
Maybe make it explicit that if no filter is supplied then ExactMatch is used as a default